src/dsl/verifier/proxy/tracer.ts

```diff
@@ -53,7 +53,7 @@ export const createResponseTracer = (): express.RequestHandler => (
       chunks.push(Buffer.from(chunk));
     }
     const body = Buffer.concat(chunks).toString('utf8');
-    logger.trace('outgoing response', removeEmptyResponseProperties(body, res));
+    logger.trace(removeEmptyResponseProperties(body, res), 'outgoing response');
```

Comment:
Oops, this is meant to be debug. I'll fix that.

Reply:
I can do it in this PR.